graphql_compiler/tests/conftest.py

```diff
@@ -35,6 +50,24 @@ def init_graph():
 
 
 @pytest.fixture(scope='class')
-def graph_client(request, init_graph):
+def graph_client(request, init_snapshot_graph_client):
     """Get a client for an initialized db, with all test data imported."""
-    request.cls.graph_client = init_graph
+    request.cls.graph_client = init_snapshot_graph_client
+
+
+@pytest.fixture(scope='class')
+def integration_graph_client(request, init_integration_graph_client):
+    """Get a client for an initialized db, with all test data imported."""
+    request.cls.graph_client = init_integration_graph_client
+
+
+@pytest.fixture(scope="class")
+def sql_integration_data(request):
+    """Generate integration data for SQL backends."""
+    sql_test_backends = init_sql_integration_test_backends()
+    generate_sql_integration_data(sql_test_backends)
+    request.cls.sql_test_backends = sql_test_backends
+    # yield the fixture to allow testing class to run
+    yield
```

Comment:
Oh interesting, I didn't realize pytest fixtures supported this.

Reply:
I didn't either, this is my first encounter with fixtures. It's pretty slick.